js-intro.js

```diff
@@ -7,37 +7,48 @@ var mantra = "Be the dev"
 
 // 1a. Write the code that determines if there is a 'B' in mantra.
 
-
+console.log(mantra.includes("B"))
 
 // 1b. Write the code that determines if there is an 'x' in mantra.
 
-
+console.log(mantra.includes("x"))
 
 // 1c. Write the code that determines if there is a 'v' in mantra.
 
-
+console.log(mantra.includes("v"))
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
+ 
+const myName = (letter) => {
+    if (letter.includes('Michael')){
+        return letter + " exists within the string.";
+    } else{
+        return letter + " does not exist within the string.";
+    }
+};
 
-
+console.log(myName('M'));
 
 // ------------------------------ Consider the following variables:
 
 var myDog = "Snoopy"
 var myCat = "Garfield"
 
+
 // 2a. Create a function that returns the longer of the two variables. Expected output: "Garfield"
+console.log(myCat)
 
```

Comment:
You are getting the correct output, but the goal is to have a function determine the length of the strings:
isLonger = (animal1, animal2) => {
  if(animal1.length < animal2.length){
    return animal2
  } else {
    return animal1
  }
}
console.log(isLonger(myDog, myCat))